apps/eth_blockchain/lib/eth_blockchain/balance.ex

```diff
@@ -65,4 +65,6 @@ defmodule EthBlockchain.Balance do
         error
     end
   end
+
+  defp do_get(_, _, _), do: {:error, :invalid_parameters}
```

Comment:
Can only reach this condition by our own coding error. Handling invalid parameters should be only for inputs entering the system. So no need for this and just let exception do its job.